test/Microsoft.ML.AutoML.Tests/Microsoft.ML.AutoML.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>Microsoft.ML.AutoML.Test</RootNamespace>
+    <ProjectGuid>{EEC3ACCD-C8BC-4261-820F-EF1340B546DE}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
```

Comment:
We use XUnit (in all .NET Foundation projects). These tests should be converted from MSTest to XUnit.

Reply:
converted.